### PR TITLE
[Urgent][Security Bugs]LPD-1558 Add captcha to avoid insufficient CSRF protection 

### PR DESCRIPTION
**Pull request by jeffwu0724**
Hi Tina,

This PR is for ticket https://liferay.atlassian.net/browse/LPD-1558.
Based on the discussion with AppSec and Security team, their expected solution for core team will be in:

- Server Administration: Script, Mail, Shutdown
- Gogo Shell

We should [enable captcha in these portlets every time](https://liferay.atlassian.net/browse/LPD-1558?focusedCommentId=2612377), and [hardcode a blacklist](https://liferay.atlassian.net/browse/LPD-1558?focusedCommentId=2569258) so that these portlets will ignore the [maxChallenges](https://liferay.atlassian.net/browse/LPD-1558?focusedCommentId=2553020) in `CaptchaConfiguration.java` that limits the captcha only show once. 

Thanks for checking.
cc: @stian-sigvartsen @topolik @ericyanLr

### CODE REVIEW

**Comment by jeffwu0724** [comment]
Screenshot:
simple captcha
image

recaptcha
image

**Comment by jeffwu0724** [comment]
ci:test:relevant

File: modules/apps/captcha/captcha-api/src/main/java/com/liferay/captcha/simplecaptcha/SimpleCaptchaImpl.java [hunk 1]
```diff
@@ -89,6 +94,20 @@ public String getTaglibPath() {
 
 	@Override
 	public boolean isEnabled(HttpServletRequest httpServletRequest) {
+		LiferayPortletConfig liferayPortletConfig =
```
Comment (jeffwu0724) [inline modules/apps/captcha/captcha-api/src/main/java/com/liferay/captcha/simplecaptcha/SimpleCaptchaImpl.java:97]: We should always returns true for these sensitive omniadmin portlet to enable captcha.
FYI, https://liferay.atlassian.net/browse/LPD-1558?focusedCommentId=2553257

**Comment by jeffwu0724** [comment]
ci:report:2553884623

**Comment by jeffwu0724** [comment]
no unique regression found

**Comment by jeffwu0724** [comment]
fixed the typo in the commit message, and the test result is still good

**Comment by brunofalc** [comment]
hey @tinatian could you prioritize this review?

**Comment by tinatian** [comment]
@brunofalc Yes, working on it.

**Pull request closed without merging**